.gitignore

```diff
@@ -8,7 +8,7 @@ __pycache__
 
 # Ignore .c files by default to avoid including generated code. If you want to
 # add a non-generated .c extension, use `git add -f filename.c`.
-*.c
+#*.c
```

Comment:
Probably best to keep the line and explicitly include the test files (since they are already included, I guess one can just revert this change).

Reply:
I re-enabled this ignore rule but added a `!` rule to specifically include _module2.c.